scripts/data/src/org/akvo/gae/remoteapi/CheckOptions.java

```diff
@@ -47,18 +50,25 @@ public void execute(DatastoreService ds, String[] args) throws Exception {
             DateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm");
             df.setTimeZone(TimeZone.getTimeZone("GMT"));
 
-            for (Entity e : pq.asIterable(FetchOptions.Builder.withChunkSize(100))) {
+            for (Entity e : pq.asIterable(FetchOptions.Builder.withChunkSize(1000))) {
                 String val = (String) e.getProperty("value");
-                if (val.startsWith("[")) {
-                    optionsJson++;
+                if (val == null) {
+                    val = ((com.google.appengine.api.datastore.Text) e.getProperty("valueText")).getValue(); //it was too big for string
+                    optionsLong++;
+                }
+                if (val == null) {
+                    optionsNull++;
```

Comment:
Can this go after L57 and we remove the condition repetition?

Reply:
Sure I can move it but the condition remains. Only if *both* fields are null should it be counted as a null value.

Reply:
Makes sense